dav1d.js

```diff
@@ -18,25 +26,46 @@ function getRuntime() {
     maximum: TOTAL_MEMORY / PAGE_SIZE,
   });
   const HEAPU8 = new Uint8Array(memory.buffer);
+  const HEAPU32 = new Uint32Array(memory.buffer);
+  HEAPU32[DYNAMICTOP_PTR >> 2] = DYNAMIC_BASE;
+
   return {
     table: table,
     memory: memory,
-    sbrk: (increment) => {
-      const oldDynamicTop = dynamicTop;
-      dynamicTop += increment;
-      return oldDynamicTop;
-    },
-    emscripten_memcpy_big: (dest, src, num) => {
+    __table_base: 0,
+    __memory_base: 1024,
+    DYNAMICTOP_PTR: DYNAMICTOP_PTR,
+    _emscripten_memcpy_big: (dest, src, num) => {
       HEAPU8.set(HEAPU8.subarray(src, src+num), dest);
     },
+    _emscripten_resize_heap: (requestedSize) => {
+      abort('OOM');
+    },
+    _emscripten_get_heap_size: () => {
+      return memory.buffer.byteLength;
+    },
     // Empty stubs for dav1d.
-    pthread_cond_wait: (cond, mutex) => 0,
-    pthread_cond_signal: (cond) => 0,
-    pthread_cond_destroy: (cond) => 0,
-    pthread_cond_init: (cond, attr) => 0,
-    pthread_cond_broadcast: (cond) => 0,
-    pthread_join: (thread, res) => 0,
-    pthread_create: (thread, attr, func, arg) => 0,
+    _pthread_cond_wait: (cond, mutex) => 0,
```

Comment:
I've used native LLVM's WebAssembly backend, seems like it doesn't require those underscores. Can you use it too?
I'm not sure which one is better though.